### PR TITLE
Initial file move.

### LINKED ISSUE #257: Move Open-CE build tools to a separate repo from the Open-CE environment files.
The Open-CE build tools and default Open-CE environment files should be kept within separate repositories, and versioned differently.

This issue should:
1. Determine the naming of the repositories.
2. Move the appropriate code to the approrpiate repositories.
3. Create plan for keeping Open-CE environment files backwards compatible with future Open-CE build tools.
4. Determine versioning scheme for Open-CE build tools.

### PR DESCRIPTION
**Pull request by bethune-bryant**
## Checklist before submitting

- [x] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/master/CONTRIBUTING.md)?
- [x] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/master/doc/)?
- [x] Did you write any [tests](https://github.com/open-ce/open-ce/blob/master/tests/) to validate this change?  

## Description

Fixes open-ce/open-ce#257

blocks open-ce/open-ce#278

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,236 @@
-# open-ce-environments
-Collection of Open-CE environment files
+
```
Comment (bnemanich) [inline README.md:2]: I feel like most of this should actually be stored with the build tools. Someone should be able to use the build tools without looking at this repo.

Comment (bethune-bryant) [reply README.md:2]: Yeah, that's a good point.
This readme  could contain a link to that document.

Maybe this readme should focus on tracking versions of packages?

Comment (bnemanich) [reply README.md:2]: Yes, I think that might be a good idea, although I'm not sure how hard that will be to maintain.

Comment (bethune-bryant) [reply README.md:2]: I think this is something we could do automatically.
I'll look into this in conjunction with open-ce/open-ce#241

**Review by bnemanich — APPROVED** [review]

**Pull request merged**